src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -345,6 +347,54 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
     }
 
     /**
+     * Loads the workflow configuration.
+     *
+     * @param array            $workflows A workflow configuration array
+     * @param ContainerBuilder $container A ContainerBuilder instance
+     * @param XmlFileLoader    $loader    An XmlFileLoader instance
+     */
+    private function registerWorkflowConfiguration(array $workflows, ContainerBuilder $container, XmlFileLoader $loader)
+    {
+        if (!$workflows) {
+            return;
+        }
+
+        $loader->load('workflow.xml');
+
+        $registryDefintion = $container->getDefinition('workflow.registry');
+
+        foreach ($workflows as $name => $workflow) {
+            $definitionDefinition = new Definition('Symfony\Component\Workflow\Definition');
+            $definitionDefinition->addMethodCall('addPlaces', [$workflow['places']]);
```

Comment:
Method `addPlaces` uses the method [addPlace](https://github.com/lyrixx/symfony/blob/component-workflow/src/Symfony/Component/Workflow/Definition.php#L88). This method requires a `Place` instance, but you will call it with an array of strings.